webapp/containers/App/App.js

```diff
@@ -30,6 +30,8 @@ class App extends PureComponent {
     super(props);
     props.loadSideNav();
     this.client = null;
+    props.hydrateClients();
+    this.boundUpdate = this.updateClient.bind(this);
```

Comment:
Are there any cases in which we need to use `updateClient` without it being bound?
If no, I think it would be better to just bind this to itself, like:
`this.updateClient = this.updateClient.bind(this)`
so that a future developer doesn't accidentally use the non-bound version because I think
you would always want the bound version

Reply:
Only bound is used. Good catch.

Reply:
updated.